CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Drone
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Drone and its packages, which are hosted in the [Drone Organization](https://github.com/drone) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+  * Comming soon!
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** [Please don't file an issue to ask a question.](http://docs.drone.io/getting-help/)
+
+## What should I know before I get started?
+
+### Drone and Plugins
+
+Drone in itself is comprised of [several projects](https://github.com/drone), [Community Plugins](https://github.com/drone-plugins) and [Public plugins](http://plugins.drone.io/)
```

Comment:
https://github.com/drone-plugins is the official plugins, and http://plugins.drone.io/ is the plugin index listing plugins and docs.

Reply:
the drone-plugins org have been given to the community, just the core plugins like docker, git and slack got limited write access.

Reply:
Ah gotcha. How does that work? We should probably document how plugins can be added.

Reply:
It's planned to write up some guidelines, but we have not done so far. Generally the plugins should be already in use and there should be some general need for the plugin, beside that it should be written in Go and have a similar structure like the other plugins :)